.rubocop.yml

```diff
@@ -0,0 +1,17 @@
+require:
+  - rubocop-rspec
+
+inherit_from:
+  - ./rubocop.yml
+  - ./rubocop-rspec.yml
+
+AllCops:
+  TargetRubyVersion: 2.5
+
+Naming/FileName:
+  Exclude:
+    - lib/rubocop-mdsol.rb
```

Comment:
This cop wants the file to be named with snake_case, but our gemspec is using `rubocop-mdsol`.
https://github.com/mdsol/rubocop-mdsol/blob/72570b0aee11d64ac9134987f4742ecd8ed5c030/rubocop-mdsol.gemspec#L4

The hyphen style also aligns nicely with other cops like below:
```yaml
require:
  - rubocop-mdsol
  - rubocop-performance
  - rubocop-rails
  - rubocop-rspec
```